### PR TITLE
Improvements in the code

### PR DESCRIPTION
**Pull request by Vkt0r**
Make improvements in the code and refactoring to improve readability and clean syntax.

### CODE REVIEW

File: AwesomeCache/Cache.swift [hunk 1]
```diff
@@ -22,6 +22,10 @@ public class Cache<T: NSCoding> {
 	private let fileManager = NSFileManager()
 	private let diskWriteQueue: dispatch_queue_t = dispatch_queue_create("com.aschuch.cache.diskWriteQueue", DISPATCH_QUEUE_SERIAL)
 	private let diskReadQueue: dispatch_queue_t = dispatch_queue_create("com.aschuch.cache.diskReadQueue", DISPATCH_QUEUE_SERIAL)
+    
+    /// Typealias to define the reusability in declaration of the closures.
+    public typealias cacheBlockClosure = (T, CacheExpiry) -> Void
+    public typealias errorClosure = (NSError?) -> Void
```
Comment (aschuch) [inline AwesomeCache/Cache.swift]: These should better start with a capital letter.

File: AwesomeCache/Cache.swift [hunk 2]
```diff
@@ -244,9 +244,7 @@ public class Cache<T: NSCoding> {
 	}
 	
 	private func sanitizedKey(key: String) -> String {
-		let regex = try! NSRegularExpression(pattern: "[^a-zA-Z0-9_]+", options: NSRegularExpressionOptions())
-		let range = NSRange(location: 0, length: key.characters.count)
-		return regex.stringByReplacingMatchesInString(key, options: NSMatchingOptions(), range: range, withTemplate: "-")
+        return key.stringByReplacingOccurrencesOfString("[^a-zA-Z0-9_]+", withString: "-", options: .RegularExpressionSearch, range: nil)
 	}
```
Comment (aschuch) [inline AwesomeCache/Cache.swift]: :heart:

**Comment by aschuch** [comment]
Thanks, just a minor remark, then I am happy to merge :)

File: AwesomeCache/Cache.swift (hunk 1 above)
Comment (Vkt0r) [inline AwesomeCache/Cache.swift]: Yeah, agree 👍

**Comment by Vkt0r** [comment]
Done @aschuch , thanks for you quickly revision 😃

**Comment by aschuch** [comment]
Thanks!

**Pull request merged**